src/common/DateField.jsx

```diff
@@ -9,7 +9,7 @@ import moment from 'moment';
 */
 const DateField = ({ handleChange, value }) => {
   const today = moment().format('YYYY-MM-DD');
-  const startOfMonth = moment().startOf('month').format('YYYY-MM-DD');
+  const minDate = moment().subtract(30, 'days').format('YYYY-MM-DD');
```

Comment:
Let store the `YYY-MM-DD` as a format somewhere because I am sure we are going to reuse it.